Basics/afw_table_guided_tour.ipynb

```diff
@@ -1102,10 +1208,11 @@
    "metadata": {},
    "source": [
     "We can see that some of the matches have very similar magnitude residuals. Look at the 6th and 7th to last row, for example:\n",
+    "\n",
```

Comment:
This is probably fragile and you might want to think about a way to avoid hard-coding the values.